lib/index.js

```diff
@@ -11,7 +11,8 @@ var Hoek = require('hoek');
 var internals = {
     initialized: false,
     defaults: {
-        logPath: Path.join(__dirname, '..', 'poop.log')
+        logPath: Path.join(__dirname, '..', 'poop.log'),
+        flags: { flags: 'a' }
```

Comment:
I agree with @devinivy, this is poorly named. I'm fine with `writeStreamOptions`. The default should be the current behavior `flags: 'w'`.